db/migrate/20200303075727_create_listing_blocked_dates.rb

```diff
@@ -0,0 +1,10 @@
+class CreateListingBlockedDates < ActiveRecord::Migration[5.2]
+  def change
+    create_table :listing_blocked_dates do |t|
+      t.references :listing
+      t.date :blocked_at
```

Comment:
Let's make [community_id, lsiting_id, blocked_at] a unique key in the DB to ensure that two entries for the same date for one listing are not possible, removing chance of race conditions during writing.